compositor_render/src/transformations/builtin/utils.rs

```diff
@@ -10,3 +11,61 @@ pub fn mat4_to_bytes(mat: &Mat4) -> bytes::Bytes {
 
     matrices_bytes.freeze()
 }
+
+pub struct BoxLayout {
```

Comment:
does not matter much, but this is one of the core concepts, I wouldn't put that in utils